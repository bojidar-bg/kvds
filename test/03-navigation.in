select 3
write X
select 1
write Y
select 2
write Z
select 21
write %
select 1
read
next
read
next
read
next
read
prev
prev
read
prev
prev
prev
read
select 12
closest
read
