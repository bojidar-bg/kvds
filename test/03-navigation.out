Y
Z
X
%
Z
Y
X
